### PR TITLE
Remove SVN support

### LINKED ISSUE #4592: Remove SVN support
**Do you want to request a *feature* or report a *bug*?**
feature

**What is the current behavior?**
GitExtensions has support for SVN

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

**What is the expected behavior?**
The very limited support for SVN is likely not of use for anyone, a dedicated client should be used.
SVN is slightly confusing in the interface.
It may not be a major maintenance issue, but still something to cleanup for 3.00

**Environment you encounter the issue:**

- GitExtensions version: master

### PR DESCRIPTION
**Pull request by gerhardol**
Fixes #4592

Changes proposed in this pull request:
 - Remove support for SVN

SVN is not a big burden, but the support in GE is very limited and a dedicated client is required to really use SVN anyway.
 
Screenshots before and after (if PR changes UI):
- Dashboard
- Start menu
- Commands

What did I do to test the code and ensure quality:
 - Started GE - SVN support is not deep in the code base

Has been tested on (remove any that don't apply):
 - Windows 10

### CODE REVIEW

**Comment by gerhardol** [comment]
No comments on the issue or the PR regarding SVN, which should come from anyone that is interested in SVN. I suggest this PR soaks at least another week before merge though.

**Comment by RussKie** [comment]
I have no problems with that.

File: GitUI/CommandsDialogs/BrowseDialog/DashboardControl/Dashboard.cs [hunk 1]
```diff
@@ -18,7 +18,6 @@ public partial class Dashboard : GitModuleControl
     {
         private readonly TranslationString _cloneFork = new TranslationString("Clone {0} repository");
         private readonly TranslationString _cloneRepository = new TranslationString("Clone repository");
-        private readonly TranslationString _cloneSvnRepository = new TranslationString("Clone SVN repository");
```
Comment (drewnoakes) [inline GitUI/CommandsDialogs/BrowseDialog/DashboardControl/Dashboard.cs]: Is there an entry in a translation file that needs to be removed as well? I haven't looked much at the translation system yet.

**Comment by RussKie** [comment]
Translations will get removed next time we regenerate translation files
before the next release.
Here's how we do it -
https://github.com/gitextensions/gitextensions/wiki/Release-cycle#2-preparing-to-release

**Pull request merged**